WordPress/Classes/ViewRelated/Pages/PageListTableViewCell.m

```diff
@@ -177,11 +166,20 @@ - (void)configureFeaturedImage
     self.labelsContainerTrailing.active = !hideFeaturedImage;
     
     if (!hideFeaturedImage) {
-        [self.featuredImageLoader loadImageFromMedia:page.featuredImage
-                                       preferredSize:CGSizeMake(FeaturedImageSize, FeaturedImageSize)
-                                         placeholder:nil
-                                             success:nil
-                                               error:nil];
+        [self.featuredImageView startLoadingAnimation];
+
+        __weak __typeof(self) weakSelf = self;
+
+        [page.featuredImage imageWithSize:CGSizeMake(FeaturedImageSize, FeaturedImageSize)
+                        completionHandler:^(UIImage * _Nullable result, NSError * _Nullable error) {
```

Comment:
I'm wondering about a case where a cell has started loading its featured image, but is then reused before the image finishes loading and starts loading a different featured image.  Depending on network latency I think we could potentially see the second image be fetched first, and then be replaced when the first image finished loading. (Does that make sense?)

Maybe we could add a check in the completion block that the cell's current page is the same as the page that had the newly acquired featured image and only set the image it if that is true?